api/src/main/java/io/opentelemetry/trace/spi/TracerFactory.java

```diff
@@ -39,4 +39,14 @@
    * @since 0.1.0
    */
   Tracer create();
```

Comment:
The [OTEP](https://github.com/open-telemetry/oteps/blob/master/text/0016-named-tracers.md) says:
> If no name (null or empty string) is specified, following the suggestions in "error handling proposal", a "smart default" will be applied and a default Tracer / Meter implementation is returned.

For my understanding, this should only serve as a fallback rather than a desired alternative to named tracers, so the parameterless `TracerFactory.create()` should be deprecated/removed.

The current [spec draft](https://github.com/open-telemetry/opentelemetry-specification/pull/264/files#diff-ea4f4a46fe8755cf03f9fb3a6434cb0cR89) also defines `name` as required with the fallback mentioned above.

Reply:
cool. will look. this was a part that was a bit confusing, since the "smart default" seems mostly to be left as an exercise for the reader. :)